README.md

```diff
@@ -1,34 +1,64 @@
 # plotjuggler-apbin-plugins
 
-This repository contains [ArduPilot Dataflash](https://ardupilot.org/copter/docs/common-logs.html) plugin for [PlotJuggler](https://github.com/facontidavide/PlotJuggler).
+This repository contains the [ArduPilot Dataflash](https://ardupilot.org/copter/docs/common-logs.html) plugin for [PlotJuggler](https://github.com/facontidavide/PlotJuggler).
 
+:construction: While you should be able to compile the plugin for Windows, this guide is tested for an Ubuntu workflow.
 
 ## Install PlotJuggler
 
 To build any plugin for PlotJuggler, PlotJuggler must be installed on your system.
-For detailled instructions on how to install PlotJuggler please have a look at the [Installation](https://github.com/facontidavide/PlotJuggler#installation) section of the PlotJuggler repository.
+For detailed instructions on how to install PlotJuggler please have a look at the [Installation](https://github.com/facontidavide/PlotJuggler#installation) section of the PlotJuggler repository.
 
 If you have ROS installed, you can install PlotJuggler using:
 
     sudo apt install ros-$ROS_DISTRO-plotjuggler-ros
 
+## (Optional) Build plotjuggler-apbin-plugin
 
-## Install plotjuggler-apbin-plugin
-The installation of the plotjuggler-apbin-plugin is straightforward.
+If you wish to build `plotjuggler-apbin-plugin` follow these steps.
+
+### 1. Clone the repository
+
+```
+git clone https://github.com/khancyr/plotjuggler-apbin-plugins
```

Comment:
This URL is outdated. This lives in ArduPilot now.

Reply:
Fixed.